src/pulse3D/excel_writer.py

```diff
@@ -307,14 +312,8 @@ def write_xlsx(
 
     data = []
 
-    twitch_width_percents = np.unique(
-        np.concatenate(
-            (
-                list(twitch_widths),
-                [(100 - width) for width in twitch_widths],
-                np.arange(10, 95, 5),
-            )
-        )
+    twitch_width_percents = tuple(
+        set([*twitch_widths, *(100 - np.array(twitch_widths, dtype=int)), *DEFAULT_TWITCH_WIDTH_PERCENTS])
```

Comment:
It's kind of like `...` in JavaScript, so it's just creating a single list of the elements from all 3 of these items since `set` only takes a single arg